design/fast_finality.md

```diff
@@ -0,0 +1,273 @@
+# Fast finality design
+
+## Scope
+
+The introduction of a single sequencer to improve cost-per-transaction during the bootstrapping phase, as 
+described in the [Bootstrapping Strategy design doc](./Bootstrapping_strategy.md).
+
+## Glossary
+
+* *Light batch*: A set of transactions produced by the designated sequencer, considered to be soft-finalised in that 
+  order
+
+## Requirements
+
+* Finality
+  * Transaction *soft* finality (finality guaranteed by the sequencer) is achieved in under one second
```

Comment:
the <1s goal is a bit ambitious. Not sure if it's achievable outside of specialised networks. We'll have to test. 
I would rephrase the goal to be: finality duration has a median of 1s.

Reply:
some analysis is needed in the design section, looking at latencies, of what should b the LB production period

Reply:
You mean, how often an LB should be produced to ensure a median of 1s?

Will update the requirement.

Reply:
I've realised I also don't say what that period covers. I guess it's from when the tx is submitted by the client.